src/base/seeds/View.svelte

```diff
@@ -106,10 +119,36 @@
       <div class="label">Version</div>
       <div>{seed.version}</div>
       <div class="desktop" />
+      <!-- User Session -->
+      <div class="label">Connection</div>
+      <div>
+        {#if $session?.signer}
+          {#if siweSession}
+            <div class="desktop">
+              <Address address={siweSession.address} {config} resolve />
+            </div>
+            <div class="mobile">
+              <Address address={siweSession.address} {config} compact resolve />
+            </div>
+          {:else}
+            <SiweConnect
+              {seed}
+              {config}
+              connection={siweSession ? Connection.Connected : Connection.Disconnected}
+            />
+          {/if}
+        {:else}
+          <div class="subtle">Please sign in to your wallet.</div>
```

Comment:
I assume this is what you mean?
```suggestion
          <div class="subtle">Please connect your wallet.</div>
```